engine/classes/Elgg/Application.php

```diff
@@ -527,33 +527,24 @@ public static function upgrade() {
 				// due to ip restrictions for example
 				if (!$rewriteTester->runLocalhostAccessTest()) {
 					// note: translation may not be available until after upgrade
-					$msg = elgg_echo("installation:htaccess:localhost:connectionfailed");
-					if ($msg === "installation:htaccess:localhost:connectionfailed") {
-						$msg = "Elgg cannot connect to itself to test rewrite rules properly. Check "
-								. "that curl is working and there are no IP restrictions preventing "
-								. "localhost connections.";
-					}
-					echo $msg;
-					exit;
-				}
-
-				// note: translation may not be available until after upgrade
-				$msg = elgg_echo("installation:htaccess:needs_upgrade");
```

Comment:
Are we OK not using elgg_echo here? I figure it's not worth it for the upgrader.